.github/workflows/aws-docker-deploy.yml

```diff
@@ -24,6 +24,11 @@ on:
         type: string
         default: ./
         description: The current working directory to run the commands in
+      tag:
+        required: false
+        type: string
+        default: ${{ github.sha }}
```

Comment:
Are you sure this works ? By this I mean using a default value that is not a constant

Reply:
I havent tested it but I would assume it works since its used in github's own checkout action https://github.com/actions/checkout/blob/main/action.yml